release/sns_notification.sh

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# HOW TO USE
+# - Prerequisites
+#   + The machine running the script must have the credentials needed to use your AWS account.
+#   + An SNS topic you wish publish the notification to. The default topic name is the value assigned to SNS_TOPIC_NAME,
+#     but you can override this if you choose. The topic must already exist, and you must have permission to list topics
+#     and publish to the intended topic. See SNS doc: https://docs.aws.amazon.com/sns/latest/dg/sns-configuring.html
+#   + A directory under NOTIFICATION_BODY_FILE_PATH that has a name that indicates what release branch you're
+#     publishing the notification about (e.g. eks-distro/tree/main/release/1-19). You must provided the directory name
+#     when running the script.
+#   + A file named 'RELEASE' in the above-mentioned release branch directory. This file must contain only the release
+#     number (e.g. "2" for 1.19.2).
+#   + A file named release-announcement.txt also in the above-mentioned release branch directory. This file must contain
+#     the intended body of the SNS message.
+# - Run ./sns_notification.sh <release branch directory name> <OPTIONAL:SNS topic name>,
+#   e.g ./sns_notification.sh 1-19 my-sns-topic
+# - When prompted to confirm that you wish to publish, enter the appropriate response to do so. All ofter response exit.
+# - If the response received from SNS did not indicate there was an error, you will be provided the SNS MessageID.
+
+RELEASE_BRANCH="${1?First required argument is release branch, e.g. 1-19}"
```

Comment:
Prow is going to be triggered by a file change, probably `release/*/release-announcement.txt` it would be a pain to parse that file name to extract the release branch so it could be recreated here. This first argument should be ANNOUNCEMNT_FILE or something of the sort.